plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -47,7 +47,7 @@ const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes
     {Swish, {{1.0f}}},
     {Mish, {{}}},
     {HSwish, {{}}},
-    {SoftPlus, {{}}}
+    {SoftPlus, {{}}},
```

Comment:
Are these openvino changes related to L0? I think we should make them a separate PR.

Reply:
This from latest plaidml-v1 branch

Reply:
I see.